packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -113,112 +106,179 @@ describe("Presence", () => {
 					});
 				});
 
-				it("is announced via `attendeeJoined` when new", () => {
-					// Act - simulate join message from client
-					presence.processSignal("", initialAttendeeSignal, false);
-
-					// Verify
-					assert(newAttendee !== undefined, "No attendee was announced");
-					assert.equal(
-						newAttendee.sessionId,
-						newAttendeeSessionId,
-						"Attendee has wrong session id",
-					);
-					assert.equal(
-						newAttendee.getConnectionId(),
-						initialAttendeeConnectionId,
-						"Attendee has wrong client connection id",
-					);
-				});
-
-				describe("disconnects", () => {
-					let disconnectedAttendee: ISessionClient | undefined;
+				describe("that joined", () => {
 					beforeEach(() => {
-						disconnectedAttendee = undefined;
 						afterCleanUp.push(
-							presence.events.on("attendeeDisconnected", (attendee) => {
-								assert(
-									disconnectedAttendee === undefined,
-									"Only one attendee should be disconnected",
-								);
-								disconnectedAttendee = attendee;
+							presence.events.on("attendeeJoined", (attendee) => {
+								assert(newAttendee === undefined, "Only one attendee should be announced");
+								newAttendee = attendee;
 							}),
 						);
-						// Setup - simulate join message from client
-						presence.processSignal("", initialAttendeeSignal, false);
-
-						// Act - remove client connection id
-						presence.removeClientConnectionId(initialAttendeeConnectionId);
 					});
+					it('is announced via `attendeeJoined`  with status "Connected" when new', () => {
+						// Act - simulate join message from client
+						presence.processSignal("", initialAttendeeSignal, false);
 
-					it("is announced via `attendeeDisconnected` when audience member leaves", () => {
 						// Verify
-						assert(
-							disconnectedAttendee !== undefined,
-							"No attendee was disconnected in beforeEach",
-						);
+						assert(newAttendee !== undefined, "No attendee was announced");
 						assert.equal(
-							disconnectedAttendee.sessionId,
+							newAttendee.sessionId,
 							newAttendeeSessionId,
-							"Disconnected attendee has wrong session id",
+							"Attendee has wrong session id",
 						);
 						assert.equal(
-							disconnectedAttendee.getConnectionId(),
+							newAttendee.getConnectionId(),
 							initialAttendeeConnectionId,
-							"Disconnected attendee has wrong client connection id",
-						);
-					});
-
-					it("changes the session client status to `Disconnected`", () => {
-						// Verify
-						assert(
-							disconnectedAttendee !== undefined,
-							"No attendee was disconnected in beforeEach",
+							"Attendee has wrong client connection id",
 						);
 						assert.equal(
-							disconnectedAttendee.getConnectionStatus(),
-							SessionClientStatus.Disconnected,
-							"Disconnected attendee has wrong status",
+							newAttendee.getConnectionStatus(),
+							SessionClientStatus.Connected,
+							"Attendee has wrong status",
```

Comment:
minor: The message could be more specific here - it could say "Attendee connection status is not Connected".
The other similar asserts don't have the luxury of nicer message since they are effectively variables.